package.json

```diff
@@ -7,9 +7,9 @@
     "build": "next build",
     "start": "next start",
     "export": "next build && next export",
-    "format": "prettier --write ./components/** ./pages/** --ignore-path ./.gitignore",
-    "lint": "eslint ./components/** ./pages/** --ext js,jsx",
-    "lint:fix": "eslint --fix ./components/** ./pages/** --ext js,jsx"
+    "format": "prettier --write \"./components/**\" \"./pages/**\" --ignore-path ./.gitignore",
+    "lint": "eslint \"./components/**\" \"./pages/**\"",
```

Comment:
Can we also add `context` directory in format and lint scripts

Reply:
Sure, I'll add this. Anything else worth including in this PR?

Reply:
Done! Formatted the files in the `context` folder as well.